### PR TITLE
changes cookie.secure default to be based on HTTP SSL

### PR DESCRIPTION
**Pull request by shawncatz**
changes cookie.secure default to depend on HTTP SSL setting
should only be enabled when SSL is enabled in revel
also, fixes name of function in comment

### CODE REVIEW

File: revel.go [hunk 1]
```diff
@@ -16,6 +16,7 @@ import (
 
 	"github.com/agtorre/gocolorize"
 	"github.com/revel/config"
+	"net/http"
```
Comment (notzippy) [inline revel.go]: I think this was added by mistake

Comment (shawncatz) [inline revel.go]: yeah, probably my IDE trying to be smart, I'll remove

Comment (shawncatz) [inline revel.go]: fixed

**Comment by shawncatz** [comment]
@notzippy can you take another look at this, I removed the unintentional additional dep

**Comment by notzippy** [comment]
Can you pull the latest from revel into this PR  @shawncatz ?

**Comment by shawncatz** [comment]
thought I already had, but must have missed something... should be up to date now...

**Pull request merged**